app/javascript/components/components/Navigation.js

```diff
@@ -1,10 +1,22 @@
 import React, { Component } from 'react'
+import { Nav, NavItem } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
 
 export default class Navigation extends Component {
     render() {
         return(
             <>
-                <h1>Navigation</h1>
+                <Nav>
+                    <NavItem>
+                        <NavLink to="/" className="nav-link">Home</NavLink>
+                    </NavItem>
+                    <NavItem>
+                        <NavLink to="/index" className="nav-link">Check Apartments</NavLink>
+                    </NavItem>
+                    <NavItem>
+                        <NavLink to="/new" className="nav-link">Log in/ Sign up</NavLink>
+                    </NavItem>
+                    </Nav>
```

Comment:
Nav can be the outermost layer here and proper React syntax would suggest it.

Reply:
thank you, i missed that part